### ISSUE TITLE
Redundant 'static lifetime for constants

### ISSUE THREAD

**Issue by dvtomas** [lukaslueg/built#9]
I just incorporated `built` into my project. It works well, but after checking it with clippy, it emits a bunch of warnings like

```
warning: Constants have by default a `'static` lifetime
 --> project/out/built.rs:7:25
  |
7 | pub const PKG_VERSION: &'static str = "0.2.0-SNAPSHOT";
  |                         ^^^^^^^ help: consider removing `'static`
  |
  = note: #[warn(const_static_lifetime)] on by default
  = help: for further information visit https://rust-lang-nursery.github.io/rust-clippy/v0.0.175/index.html#const_static_lifetime

warning: Constants have by default a `'static` lifetime
 --> /home/td/projects/wearcomputer-proc/target/debug/build/wearcomputer-proc-7427d4ac96b4c4d9/out/built.rs:9:31
  |
9 | pub const PKG_VERSION_MAJOR: &'static str = "0";
  |                               ^^^^^^^ help: consider removing `'static`
  |
  = help: for further information visit https://rust-lang-nursery.github.io/rust-clippy/v0.0.175/index.html#const_static_lifetime

.
.
.
```

It indeed looks like the `'static` specifier is redundant and could be removed to make clippy happier.

**Comment by lukaslueg** [comment]
Indeed, `'static` on `const` became redundant in 1.19ish. A PR would be welcome, I'll fix it up soon otherwise.

**Comment by lukaslueg** [comment]
Fixed in 6628ed531444bfa4c30826cb987b1a9d64fa87a6
